regulations/static/regulations/js/source/views/header/preamble-head-view.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var PreambleHeadView = Backbone.View.extend({
+  el: '.preamble-header',
+
+  events: {
+    'click .read-proposal': 'readProposal',
+    'click .write-comment': 'writeComment'
+  },
+
+  initialize: function() {
+    this.$read = $('#preamble-read');
+    this.$write = $('#preamble-write');
+    this.$readTab = this.$el.find('.read-proposal');
+    this.$writeTab = this.$el.find('.write-comment');
+
+    Backbone.on('writeSectionComment', this.writeComment, this);
+  },
+
+  readProposal: function() {
+    this.$readTab.addClass('active-mode');
+    this.$writeTab.removeClass('active-mode');
+
+    Backbone.trigger('readProposal');
+  },
+
+  writeComment: function() {
+    this.$writeTab.addClass('active-mode');
+    this.$readTab.removeClass('active-mode');
```

Comment:
I'm thinking we'll emit an event here that we're listening for in `PreambleView`--something like `CommentEvents.trigger('comment:write')`. We can listen in `PreambleView` and call `this.write` with the appropriate section.